Makefile

```diff
@@ -29,3 +29,8 @@ generate-dockerfiles:
 generate-release:
 	./openshift/ci-operator/generate-release.sh $(BRANCH) > release.yaml
 .PHONY: generate-release
+
+# Generate an aggregated knative yaml file with replaced image references
+resolve-knative:
+	./openshift/release/resolve.sh config/ openshift/knative-serving-$(RELEASE).yaml "quay.io/openshift-knative/knative-serving-" $(RELEASE)
```

Comment:
Should this go under `openshift/olm/knative-serving-$(RELEASE)/knative-serving-$(RELEASE).yaml`? So we can have a folder for all the OLM manifests needed for each release?

Reply:
What about just `openshift/olm/knative-serving-$(RELEASE).yaml` ?

Reply:
It really has nothing to do with olm, though. I almost feel like it should go in `config/` or wherever build artifacts normally go.

Reply:
When the operator is part of the fork, i.e. upstream, I know exactly where it should go: the equivalent of its `deploy/resources`

Reply:
Do the upstream release YAMLs not only live in their releases (attached to the github release)?

Reply:
I think they're only available from the github releases page @markusthoemmes